public/sitemap.xml

```diff
@@ -107,7 +107,7 @@
 <url><loc>https://doc.babylonjs.com/contribute/contributeToDocs/easydoc</loc><lastmod>2022-12-13T12:45:57.562Z</lastmod></url>
 <url><loc>https://doc.babylonjs.com/contribute/contributeToDocs</loc><lastmod>2022-10-17T11:20:30.293Z</lastmod></url>
 <url><loc>https://doc.babylonjs.com/contribute/toBabylon/HowToContribute</loc><lastmod>2022-12-13T12:45:57.563Z</lastmod></url>
-<url><loc>https://doc.babylonjs.com/contribute/toBabylon/Prerequisits</loc><lastmod>2022-09-22T12:30:30.282Z</lastmod></url>
+<url><loc>https://doc.babylonjs.com/contribute/toBabylon/Prerequisites</loc><lastmod>2022-09-22T12:30:30.282Z</lastmod></url>
```

Comment:
sitemap is updated on a weekly base in an external process. You can revert this change

Reply:
Done. BTW: Is the sitemap update process documented anywhere? I can't seem to find any info about it in the docs.

I'm just curious, but I guess it might be worth documenting if it's something that contributors would need to know about.

Reply:
it's not documented intentionally :-)

But running locally `npm run dev:local` will generate the sitemap